app/services/foreman_ansible/fact_parser.rb

```diff
@@ -72,6 +76,11 @@ def ip_from_interface(interface)
       facts[:"ansible_#{interface}"]['ipv4']['address']
     end
 
+    def ipv6_from_interface(interface)
+      return if facts[:"ansible_#{interface}"]['ipv6'].blank?
```

Comment:
this will now break on rubocop, new line is required after return

Reply:
Hi @ares , thank you for the review! Will do the changes later today

Reply:
thanks for letting me know, ok, I'll wait with the new release then

Reply:
Hi @ares, didn't want to be the blocking factor so changed it right away :-)

Reply:
Thanks! I'll give it a short test tomorrow and will merge and release 2.3 if everything goes right!